.github/workflows/crowdin-action.yml

```diff
@@ -1,3 +1,6 @@
+# FIXME ENG-14 - I can't see this running anywhere
```

Comment:
@camerow: I added some notes here as I don't think this is doing anything now. We should probably set `upload_translations: true`? 

We could do better reverting to a checked in `.po` file as now:
- crowdin creates new `id`s added in code with an empty translation
- in the production build it loads the translations from CrowdIn and has no backup
- if a new key is added and CrowdIn not updated it will render the key in the UI e.g. `my.key.id` 

The production app should have a fallback `.po` file that we check in and can also then use for when we are testing. It's unclear what will happen in the production build if it cannot connect to CrowdIn